rflx/pyrflx/utils/checksum_functions.py

```diff
@@ -0,0 +1,16 @@
+def internet_checksum(checksum_bytes: bytes) -> int:
```

Comment:
```
from rflx.pyrflx.utils.checksum_functions import internet_checksum
```

I would prefer a shorter version by simply putting `internet_checksum` in `utils`.